simulator/setting_up_simulator.md

```diff
@@ -48,50 +62,49 @@ The simulation for the SR2024 competition will be released at Kickstart.
     ```
     If there is not an existing `robot.py` an example one will be created when the simulator first runs.
 
-4. Using the Webots IDE, open the `worlds/Arena.wbt` file.
+4. Open the Webots IDE, then use that to open the `worlds/Arena.wbt` file.
 
 You may receive a warning about your computer's GPU not being good enough, which can be ignored.
 
 -->
 
 <div class="info">
-  On recent versions of macOS you may need to give Webots permission to access the directory where you have extracted the simulation files.
+On recent versions of macOS you may need to give Webots permission to access the directory where you have extracted the simulation files.
 </div>
 
+
 ### Changing your version of Python
 
-If webots is not picking any version of Python or is picking up the wrong one then you'll need to change it.
+Sometimes Webots will not automatically detect your installed Python so we need to change it.
 When this happens Webots will print errors to its console and your robot will not move.
 
 You will need the full path to the version of Python that you want to use.
 This will vary based on the system you have.
-One way to find the path is by launching the version of Python that you want to
-use and running the following code:
+One way to find the path is by launching Python and running the following code:
 
 ~~~~~ python
 import sys
 print(sys.executable)
 ~~~~~
 
-On Windows you can set the path to the Python version to use in Webots UI via
-**Tools** &rarr; **Preferences** &rarr; **General** &rarr; **Python command**.
+Once you have the path you need to enter this into the Webots settings.
```

Comment:
:+1: to adding this comment. It's probably a good idea to separate it from the following paragraphs though (otherwise it'll run into the Windows one)
```suggestion
Once you have the path you need to enter this into the Webots settings.

```

Reply:
1b1c6a7